src/SFA.DAS.EmployerIncentives.Web.SystemAcceptanceTests/HttpClientExtensions.cs

```diff
@@ -1,20 +1,20 @@
-﻿using Newtonsoft.Json;
+﻿using System.Collections.Generic;
 using System.Net.Http;
-using System.Text;
 using System.Threading.Tasks;
 
 namespace SFA.DAS.EmployerIncentives.Web.SystemAcceptanceTests
 {
     public static class HttpClientExtensions
     {
 
-        public static async Task<HttpResponseMessage> PostValueAsync<T>(this HttpClient client, string url, T data)
+        public static async Task<HttpResponseMessage> PostFormAsync(this HttpClient client, string url, params KeyValuePair<string, string>[] data)
```

Comment:
Just wondering if this should an extra method rather than replacing the existing PostValueAsync, as we might still want to post JSON

Reply:
we might but shouldn't if we using MVC controllers.
but we can always bring it back, I've incorrectly added `PostValueAsync` I shouldn't have used in my tests and now it's replaced and renamed for clarity.